mu/locale/cs/LC_MESSAGES/mu.po

```diff
@@ -0,0 +1,1866 @@
+msgstr ""
+"Nepodařilo se najít localhost.\n"
+"Ujistěte se, že máte záznam '127.0.0.1 localhost' v souboru /etc/hosts."
+
+#: mu/interface/dialogs.py:76
+msgid "Select Mode"
+msgstr "Zvolte režim"
+
+#: mu/interface/dialogs.py:79
+msgid ""
+"Please select the desired mode then click \"OK\". Otherwise, click "
+"\"Cancel\"."
+msgstr ""
+"Zvolte, prosím, požadovaný režim a potom klikněte na tlačítko \"OK\". V "
+"opačném případě klikněte na tlačítko \"Zrušit\"."
+
+#: mu/interface/dialogs.py:100
+msgid ""
+"Change mode at any time by clicking the \"Mode\" button containing Mu's "
+"logo."
+msgstr ""
+"Režim můžete kdykoliv změnit kliknutím na tlačítko \"Režim\" s logem "
+"editoru Mu."
+
+#: mu/interface/dialogs.py:139
+msgid ""
+"When reporting a bug, copy and paste the content of the following log "
+"file."
+msgstr "Při nahlašování chyby zkopírujte a vložte obsah tohto záznamu."
+
+#: mu/interface/dialogs.py:163
+msgid ""
+"The environment variables shown below will be set each time you run a "
+"Python 3 script.\n"
+"\n"
+"Each separate enviroment variable should be on a new line and of the "
+"form:\n"
+"NAME=VALUE"
+msgstr ""
+"Proměnné prostředí, které se nachází níže, budou nastavené vždy, když "
+"spustíte Python 3 skript.\n"
+"\n"
+"Každá proměnná prostředí by se měla nacházet na novém řádku ve formátu:\n"
+"NÁZEV = HODNOTA"
+
+#: mu/interface/dialogs.py:189
+msgid "Minify Python code before flashing?"
+msgstr "Minifikovat Python kód před jeho zapsáním do paměti?"
+
+#: mu/interface/dialogs.py:193
+msgid ""
+"Override the built-in MicroPython runtime with the following hex file "
+"(empty means use the default):"
+msgstr ""
+"Přepsat vestavěný MicroPython následujícim binárním soborem (prázdné pole "
+"znamená, že se použije výchozí nastavení):"
+
+#: mu/interface/dialogs.py:219
+msgid ""
+"The packages shown below will be available to import in Python 3 mode. "
+"Delete a package from the list to remove its availability.\n"
+"\n"
+"Each separate package name should be on a new line. Packages are "
+"installed from PyPI (see: https://pypi.org/)."
+msgstr ""
+"Níže zobrazené balíčky bude možné importovať v režimu Python 3. "
+"Odstraněním balíčku ze seznamu zamezíte jeho dostupnosti.\n"
+"\n"
+"Název každého samostatného balíčku by se měl nacházet na novém řádku. "
+"Balíčky jsou instalované z PyPI (viz: https://pypi.org/)."
+
+#: mu/interface/dialogs.py:250
+msgid ""
+"The folks at <a href='https://www.pythonanywhere.com/'>PythonAnywhere</a>"
+" make it easy for learners and educators to host simple web projects for "
+"free. You'll need to sign up for an account and provide the following "
+"details for Mu to deploy your web project."
+msgstr "Parta na <a href='https://www.pythonanywhere.com/'>PythonAnywhere</a> pracuje na tom, aby začátečníci a učitelé mohli zdarma provozovat jednoduché projekty na internetu. Budete si tam muset vytvořit účet a zadat následující údaje, aby Mu mohlo váš projekt zprovoznit."
```

Comment:
"Parta" mi nepřijde příliš profesionální, ale to není ani "folks". Nic lepšího mě nenapadá. :/